lua/treesitter-context/render.lua

```diff
@@ -386,6 +386,34 @@ end
 
 local M = {}
 
+-- Contexts may sometimes leak due to reasons like the use of 'noautocmd'.
```

Comment:
I'm not sure we should be putting in provisions for handling noautocmd cases. If a bug is caused by noautocmd, then it means it was used incorrectly.